testnets/xiontestnet/chain.json

```diff
@@ -0,0 +1,120 @@
+{
+  "$schema": "../../chain.schema.json",
+  "chain_name": "xion-testnet-1",
+  "status": "live",
+  "network_type": "testnet",
+  "pretty_name": "Xion Testnet",
+  "chain_id": "xion-testnet-1",
+  "bech32_prefix": "xion",
```

Comment:
Please register this prefix: https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
Thank you for pointing that out, we've opened a PR there.

https://github.com/satoshilabs/slips/pull/1531

Reply:
will set this PR as ready for review when the PR to satoshilabs goes through